OMCompiler/Compiler/runtime/printimpl.c

```diff
@@ -94,14 +94,14 @@ static void make_key()
 
 static print_members* getMembers(threadData_t *threadData)
 {
-#if defined(OMC_NO_THREADS)
+#if !defined(OM_HAVE_PTHREADS)
   static
 #endif
   print_members *res = NULL;
   if (threadData && threadData->localRoots[LOCAL_ROOT_PRINT_MO]) {
     return threadData->localRoots[LOCAL_ROOT_PRINT_MO];
   }
-#if !defined(OMC_NO_THREADS)
+#if defined(OM_HAVE_PTHREADS)
```

Comment:
This way around is so much better!

Reply:
It is. Unfortunately though, the Makefiles build is turning out to be problematic because there is no one place to define this. I taught it was enough to define in when compiling the runtime libraries related to PThreads. However, the header files are included directly or indirectly all over the codebase and I have to handle them everywhere. Adding it to the `configure.ac` helps but it won't solve everything since there are places that are not "configured" at all and then there is also Windows which has its own static Makefiles.  I am trying to fix them one by one.